nodejs/awsx/apigateway/api.ts

```diff
@@ -649,8 +732,15 @@ function addStaticRouteToSwaggerSpec(
                     if (childPath === indexPath) {
                         // We hit the file that we also want to serve as the index file. Create
                         // a specific swagger path from the server root path to it.
+                        let swaggerOperation = createSwaggerOperationForObjectKey(childUrn, role);
+                        if (directory.requiredParameters) {
+                            addRequiredParametersToSwaggerOperation(swaggerOperation, directory.requiredParameters);
```

Comment:
it might be worth documenting that for a static route, the required parameters will apply to all of them. 

Note: it's also not clear to me what it means for a static route to even have validation.  It's not like the query variables would ever be involved in anything right?

Reply:
Will do 

and yeah -- I was a bit unsure if I should to include them or not for static routes for this reason.